core/templates/pages/splash-page/splash-page.component.html

```diff
@@ -1,174 +1,354 @@
+    </div>
+    <div ng-if="$ctrl.testimonialId === 2">
+      <h2 translate="I18N_SPLASH_TESTIMONIAL_3"></h2>
+      <h3 translate="I18N_SPLASH_STUDENT_DETAILS_3"></h3>
     </div>
+    <span ng-click="$ctrl.decrementTestimonialId()" class="material-icons">
+      arrow_back_ios
+    </span>
+    <span ng-click="$ctrl.incrementTestimonialId()" class="material-icons">
+      arrow_forward_ios
+    </span>
+  </div>
+  <div class="oppia-splash-section-five-right" ng-if="!$ctrl.isWindowNarrow()">
+    <img ng-if="$ctrl.testimonialId === 0" class="splash-student-image" ng-src="<[$ctrl.getStaticImageUrl('/splash/mira.png')]>" alt="">
+    <img ng-if="$ctrl.testimonialId === 1" class="splash-student-image" ng-src="<[$ctrl.getStaticImageUrl('/splash/sama.png')]>" alt="">
+    <img ng-if="$ctrl.testimonialId === 2" class="splash-student-image" ng-src="<[$ctrl.getStaticImageUrl('/splash/sara.png')]>" alt="">
+  </div>
+</div>
+
+<div class="oppia-splash-section-six">
+  <div class="oppia-splash-section-six-left">
+    <img ng-src="<[$ctrl.getStaticImageUrl('/splash/books_dsk.png')]>" class="splash-books-image" alt="">
+  </div>
+  <div class="oppia-splash-section-six-right">
+    <h3 translate="I18N_SPLASH_FOR_TEACHERS"></h3>
+    <h2 translate="I18N_SPLASH_TEACHERS_TITLE"></h2>
+    <p translate="I18N_SPLASH_TEACHERS_CONTENT"></p>
+    <div class="oppia-splash-button-container">
+      <form action="/teach">
+        <button type="submit"
+                class="btn oppia-splash-button oppia-splash-button-browse oppia-splash-browse-button-position oppia-transition-200"
+                translate="I18N_SPLASH_LEARN_MORE">
+        </button>
+      </form>
+    </div>
+  </div>
+</div>
+
+<div class="oppia-splash-section-seven" ng-style="{'background-image': '<[$ctrl.getCommunityBackgroundUrl()]>'}">
+  <div class="oppia-splash-section-seven-image" ng-if="$ctrl.isWindowNarrow()">
+    <img ng-src="<[$ctrl.getStaticImageUrl('/splash/volunteer_section.png')]>" class="splash-vol-image" alt="">
+  </div>
+  <div class="oppia-splash-section-seven-left">
+    <h3 translate="I18N_SPLASH_FOR_VOLUNTEERS"></h3>
+    <h2 translate="I18N_SPLASH_VOLUNTEERS_TITLE"></h2>
+    <p translate="I18N_SPLASH_VOLUNTEERS_CONTENT"></p>
+    <div class="oppia-splash-button-container">
+      <form action="https://www.oppiafoundation.org/volunteer">
+        <button type="submit"
+                class="btn oppia-splash-button oppia-splash-button-browse oppia-splash-browse-button-position oppia-transition-200"
+                translate="I18N_SPLASH_START_CONTRIBUTING">
+        </button>
+      </form>
+    </div>
+  </div>
+  <div class="oppia-splash-section-seven-right" ng-if="!$ctrl.isWindowNarrow()">
+    <img ng-src="<[$ctrl.getStaticImageUrl('/splash/volunteer_section.png')]>" class="splash-vol-image" alt="">
   </div>
 </div>
 
 <style>
-  splash-page .oppia-splash-rows-container {
+  splash-page .oppia-splash-section-four {
```

Comment:
[Optional] Using tag name in CSS selectors doesn't work out in Angular. Can you wrap this entire page in a div of class `splash page` and change the selector to `.splash-page .oppia-splash-section-four`. And similarly at other places.

Reply:
Done